setup.cfg

```diff
@@ -15,6 +15,7 @@ exclude =
     apptools/type_manager/*
     apptools/type_registry/*
     apptools/undo/*
+    docs/source/conf.py
```

Comment:
(I found this because I did not use etstool.py, I just used a Python venv with flake8 installed... so naturally I just did `flake8 .` and errors for this file turn up.)